package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "react-date-time-group",
+  "version": "1.0.0",
+  "description": "Datepicker with optional time",
+  "main": "dist/DateTimeGroup.js",
+  "scripts": {
+    "ci": "npm run lint && npm run coverage",
+    "coverage": "istanbul cover _mocha -- --compilers js:babel/register --require react-tests-globals-setup test/test* && istanbul check-coverage --statements 95 --branches 95 --functions 95 --lines 95",
```

Comment:
this `--statements 95 --branches 95 --functions 95 --lines 95` can go into .istanbul.yml